src/aws/Dockerfile

```diff
@@ -41,6 +109,18 @@ RUN cd lambda-migrate && yarn install
 
 CMD [ "index.handler" ]
 
+# Image used by circleci to run tests
+FROM development as circleci
+
+## Install packages needed to circle-ci
+RUN yum install -y \
+    bash \
+    git \
+    openssh \
+    tar \
+    gzip \
+    ca-certificates
```

Comment:
The jot is not run using this image. You use a machine one and then you use directly the docker CLI. I'm pretty sure this stage is not needed.